integration-tests/pom.xml

```diff
@@ -310,6 +310,7 @@
                 <module>micrometer-mp-metrics</module>
                 <module>micrometer-prometheus</module>
                 <module>opentelemetry</module>
+                <module>opentelemetry-jdbc-instrumentation</module>
```

Comment:
AFAICS, the original issue was with native executables and I don't see the module added here: https://github.com/quarkusio/quarkus/blob/main/.github/native-tests.json ?

Is it an oversight or your intention?

Reply:
I forgot, it should be fixed now (I'll check CI), thanks.